pkg/servers/servers_test.go

```diff
@@ -89,54 +89,55 @@ var _ = AfterSuite(func() {
 
 var _ = Describe("HTTPListeners", func() {
 	It("starts http only server", func() {
-		listeners := New("8080", "", "", "")
+		listeners := New("8080", "", "", "", true)
 
 		Expect(listeners.HTTP).NotTo(BeNil())
 		Expect(listeners.HTTP.Addr).To(Equal(":8080"))
 		Expect(listeners.HTTPS).To(BeNil())
 		Expect(listeners.HasBothHandlers).To(BeFalse())
 
 		listeners.ListenAndServe()
-		timeout := time.Second
-		_, err := net.DialTimeout("tcp", net.JoinHostPort("localhost", "8080"), timeout)
-		Expect(err).NotTo(HaveOccurred())
+		Eventually(func() error {
+			_, err := net.DialTimeout("tcp", "localhost:8080", 100*time.Millisecond)
```

Comment:
Lets add `100*time.Millisecond` as a const. Improving the connection setup in unittests up to 10 is good, lets see how far can we push it?